exercises/practice/pascals-triangle/.meta/config.json

```
@@ -1,6 +1,13 @@
 {
   "blurb": "Compute Pascal's triangle up to a given number of rows.",
-  "authors": [],
+  "authors": [
+    "jvarness"
+  ],
+  "contributors": [
+    "devkabiir",
+    "ErikSchierboom",
+    "Stargator"
+  ],
```

Leaving the following contributors due to substantial refactoring of the example implementation during bulk PR #306
```suggestion
  "contributors": [
    "devkabiir",
    "Stargator"
  ],
```